data/src/main/java/co/nimblehq/compose/crypto/data/model/response/CoinItemResponse.kt

```diff
@@ -30,9 +35,41 @@ data class CoinItemResponse(
     val marketCapChange24h: BigDecimal?,
     @Json(name = "market_cap_change_percentage_24h")
     val marketCapChangePercentage24h: Double?,
+    @Json(name = "circulating_supply")
+    val circulatingSupply: BigDecimal?,
+    @Json(name = "total_supply")
+    val totalSupply: BigDecimal?,
+    @Json(name = "max_supply")
+    val maxSupply: BigDecimal?,
+    @Json(name = "ath")
+    val ath: BigDecimal?,
+    @Json(name = "ath_change_percentage")
+    val athChangePercentage: BigDecimal?,
+    @Json(name = "ath_date")
+    val athDate: String?,
+    @Json(name = "atl")
+    val atl: BigDecimal?,
+    @Json(name = "atl_change_percentage")
+    val atlChangePercentage: BigDecimal?,
```

Comment:
same

Reply:
@luongvo Please consider some response items:

<img width="752" alt="Screen Shot 2565-09-07 at 14 46 09" src="https://user-images.githubusercontent.com/28002315/188820103-5b291de3-b196-416f-be5c-c6465654cdf4.png">

Reply:
@Wadeewee Seems we don't use these fields on the Integration part for the Trending list?

@luongvo I think we could consider using Decimal as well since the percentage might be a large number.

Reply:
> Seems we don't use these fields on the Integration part for the Trending list?

@hoangnguyen92dn  Did you mean for the `roi` object?

Reply:
@hoangnguyen92dn @Wadeewee using BigDecial or Double depends on should we keep the precision on number calculating https://stackoverflow.com/questions/3413448/double-vs-bigdecimal. IMO, we don't need to use it for percentage. BigDecimal is slower and bigger than Double. Double is big enough to handle these values I believe.